src/Groups/GAPGroups.jl

```diff
@@ -994,13 +1046,17 @@ the intersection of all maximal subgroups of `G`.
 @gapattribute frattini_subgroup(G::GAPGroup) = _as_subgroup(G, GAP.Globals.FrattiniSubgroup(G.X))
 
 """
-    radical_subgroup(G::GAPGroup)
+    solvable_radical(G::GAPGroup)
 
 Return the solvable radical of `G`, i.e.,
 the largest solvable normal subgroup of `G`.
 """
-@gapattribute radical_subgroup(G::GAPGroup) = _as_subgroup(G, GAP.Globals.RadicalGroup(G.X))
-#T wrong name, already in GAP!
+@gapattribute solvable_radical(G::GAPGroup) = _as_subgroup(G, GAP.Globals.SolvableRadical(G.X))
+
+@deprecate radical_subgroup solvable_radical
+@deprecate has_radical_subgroup has_solvable_radical
+@deprecate set_radical_subgroup set_solvable_radical
```

Comment:
I think there is a file where we collect them?

Reply:
Indeed, `src/Deprecations.jl`

We also have some in `src/Groups/sub.jl` (I think they may predate `src/Deprecations.jl`)